lib/jekyll/entry_filter.rb

```diff
@@ -123,5 +123,11 @@ def glob_include?(enum, entry)
         end
       end
     end
+
+    private
+
+    def source_path
```

Comment:
:thumbsup:

reduces allocations and assigns a name to a concept in this class that was implicit before. thanks!